src/app/assets/languages/en.json

```diff
@@ -137,16 +137,18 @@
       "create-button-label": "Create",
       "back-button-label": "Back",
       "getting-started": {
-        "page-title": "Getting started",
-        "cache-name": "1. Give a name to your cache",
-        "cache-type": "2. What do you want to do?",
-        "cache-type-radio1": "Configure a new cache",
-        "cache-type-radio1-helper": "Description of Configure a new cache",
-        "cache-type-radio2": " Use editor to config a cache",
-        "cache-type-radio2-helper": "Description of Use editor to config a cache"
+        "nav-title": "Get started",
+        "page-title": "Start with the basics",
+        "cache-name": "1. Enter a unique name for your cache.",
+        "cache-type": "2. Choose how to create your cache.",
+        "cache-type-radio1": "Create a new cache",
+        "cache-type-radio1-helper": "Build a cache with guided steps.",
+        "cache-type-radio2": "Add an existing cache",
+        "cache-type-radio2-helper": "Provide valid cache configuration."
       },
       "edit-config":{
-        "page-title": "Edit code"
+        "nav-title": "Configure ",
```

Comment:
@oraNod nav-title is mentioned "Edit Code" on marvel app (https://marvelapp.com/prototype/2b2cgifg/screen/81483820) , Could you please confirm

Reply:
@dipguptaredhat Hey, I don't think we need to strictly follow the wording in the marvel app. I do see that I left out an update in my commit and that string is wrong. I'll fix it now. Thanks for pointing it out and asking.

Reply:
Just pushed another commit to update the wording. I think we'll need to come back to the text strings anyway so let's not regard these as final yet.